src/components/button/Button.scss

```diff
@@ -0,0 +1,26 @@
+.button {
+  height: 100px;
+  width: 90px;
+  border-radius: 7px;
+  background-color: gray;
+  display: flex;
+  flex-direction: column;
+  justify-content: center;
+  align-items: center;
+  cursor: pointer;
```

Comment:
can remove this since once we convert them to actual buttons, the cursor will automatically be pointer